.github/workflows/python-test-conda.yml

```diff
@@ -16,36 +16,107 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ["3.8"]
+        python-version: ["3.6", "3.10"]
+        # Breaks due to https://github.com/jpype-project/jpype/issues/1009
+        # Should be included once the fix is released
+        exclude:
+          - os: windows-latest
+            python-version: "3.10"
+        include:
+          - os: windows-latest
+            python-version: "3.9"
 
     steps:
     - uses: actions/checkout@v2
-    - uses: conda-incubator/setup-miniconda@v2
+
+    - uses: actions/setup-python@v3
       with:
-        auto-update-conda: true
-        python-version: ${{ matrix.python-version }}
-    - name: Add conda to system path
-      run: |
-        # $CONDA is an environment variable pointing to the root of the miniconda directory
-        echo $CONDA/bin >> $GITHUB_PATH
-    - name: Install mamba
-      run: |
-        conda install -c conda-forge mamba
-    - name: Install dependencies
-      run: |
-        mamba env update --file environment-test.yml --name base
-    - name: Install current project in dev mode
-      run: |
-        pip install -e .
-    - name: Install pytest
-      run: |
-        mamba install -c conda-forge pytest
-    - name: Test ImageJ2 + original ImageJ
-      run: |
-        pytest test/ -p no:faulthandler
-    - name: Test ImageJ2 standalone
+        python-version: ${{matrix.python-version}}
+
+    - uses: actions/setup-java@v3
+      with:
+        java-version: '8'
+        distribution: 'zulu'
+
+    - name: Install pyimagej
       run: |
-        pytest test/ -p no:faulthandler --legacy=false
-    - name: Test Fiji Is Just ImageJ(2)
+        python -m pip install --upgrade pip
+        python -m pip install -e ".[test]"
+
+    - name: Test ImageJ
       run: |
-        pytest test/ -p no:faulthandler --ij=sc.fiji:fiji
+        ./test.sh
```

Comment:
Yay, I'm glad we can just call this shell script now.

Reply:
:confetti_ball: